Documentation/Upgrade/README.rst

```diff
@@ -0,0 +1,105 @@
+
+
+Basic Configuration
+---------------------
+
+The basic configuration includes:
+   
+   * An existing installed VSD and VSC upgraded to latest VSP release
+   * A single OpenStack Controller node
+   * One or more Compute nodes (hypervisors) running the VRS, SR-IOV, or AVRS nodes with the latest VSP release
+   
+
+
+Before the Upgrade
+--------------------
+
+1. If you are upgrading an AVRS node, migrate all your VMs on the node to an AVRS Compute node that is not being upgraded. Perform the steps in the "Live Migrate a Virtual Machine" section in https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/migrating-virtual-machines-between-compute-nodes-osp#live-migrate-a-vm-osp
+
+2. Create the following repositories:
+    
+    * Nuage_base_repo:
+        
+        - It contains Selinux-policy-nuage, nuage-bgp, Python-openvswitch-nuage, nuage-openstack-neutronclient, and nuage-puppet-modules.
+        - Enable this repository on the OpenStack Controller nodes, the Compute VRS or AVRS nodes, and the Nova Compute AVRS nodes.
+    
+    * Nuage_vrs_repo:
+        
+        - It contains nuage-metadata and nuage-openvswitch.
+        - Enable this repository on the OpenStack Controller node and the Nova Compute VRS nodes.
+        
+    
+    * Nuage_avrs_repo:
+        
+        - It contains nuage-metadata, nuage-openvswitch, nuage-selinux-avrs, and 6WIND packages.
+        - Enable this repository on the Nova Compute AVRS nodes.
+        
+    
+
+
+3. Get the latest Nuage docker images from the Red Hat Partner Registry by following these instructions: https://github.com/nuagenetworks/nuage-ospdirector/blob/OSPD13/Documentation/5.4.1/README.rst#phase-8-nuage-docker-containers
+
+Upgrade Workflow
+------------------
+
+1. Back up the configuration files for your deployment.
+    
+    In the following examples, the templates and environment files are in the /home/stack/nuage-ospdirector directory.
+    
+    a. Back up the templates and environment files from /home/stack/nuage-ospdirector to /home/stack/nuage-ospdirector-bk.
+       
+       ::
+       
+           $ mv /home/stack/nuage-ospdirector /home/stack/nuage-ospdirector-bk
+       
+    
+    
+    b. Download the tar files for the upgrade from https://github.com/nuagenetworks/nuage-ospdirector/releases
+    
+
+2. Make sure your all of the templates and environment files are updated with the environment values for your deployment.
+
+    Get the environment values from the /home/stack/nuage-ospdirector-bk directory and update all the templates and environment files for the deployment, such as neutron-nuage/nova-nuage/compute-avrs and templates/compute-avrs-role.yaml.
+
+
+3. To update the Overcloud image, follow these instructions: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/keeping_red_hat_openstack_platform_updated/assembly-updating_the_overcloud

```

Comment:
here we have steps to update overcloud deployments, updating overcloud image comes in step 4

Reply:
done